app/src/main/res/values/dimens.xml

```diff
@@ -1,10 +1,25 @@
 <resources>
-    <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
-    <dimen name="fab_margin">16dp</dimen>
-    <dimen name="app_bar_height">200dp</dimen>
-    <dimen name="item_width">200dp</dimen>
-    <dimen name="text_margin">16dp</dimen>
-    <dimen name="spacing_medium">16dp</dimen>
+
+    <dimen name="margin_extra_small">4dp</dimen>
```

Comment:
@iamareebjamal @arp95 I have created separate padding and margin dimes intentionally. Both are very different things and will give different results for same values. 

for example - later we may change our padding scale. Like we may change `padding_normal` to `18dp` keeping `margin_normal` to `16dp`.

Hence keeping them separate is good idea and also its not gonna make any performance issues.;p